cli/azd/cmd/cobra_builder.go

```diff
@@ -188,7 +188,6 @@ func (cb *CobraBuilder) bindCommand(cmd *cobra.Command, descriptor *actions.Acti
 
 	// Create, register and bind flags when required
 	if descriptor.Options.FlagsResolver != nil {
-		log.Printf("registering flags for action '%s'\n", actionName)
```

Comment:
We can always revisit this later, but I removed this since it hasn't been that useful